src/components/views/messages/EditHistoryMessage.tsx

```diff
@@ -59,11 +59,11 @@ export default class EditHistoryMessage extends React.PureComponent<IProps, ISta
         super(props);
 
         const cli = MatrixClientPeg.get();
-        const { userId } = cli.credentials;
+        const userId = cli.getSafeUserId();
         const event = this.props.mxEvent;
         const room = cli.getRoom(event.getRoomId());
         if (event.localRedactionEvent()) {
-            event.localRedactionEvent().on(MatrixEventEvent.Status, this.onAssociatedStatusChanged);
+            event.localRedactionEvent()!.on(MatrixEventEvent.Status, this.onAssociatedStatusChanged);
```

Comment:
The `if` condition can be replaced by

```suggestion
            event.localRedactionEvent()?.on(MatrixEventEvent.Status, this.onAssociatedStatusChanged);
```

(same for other if below)